styles.css

```diff
@@ -458,3 +458,167 @@ ul {
   text-align: center;
   color: #fff;
 }
+
+/* new branch css stats here */
+form {
+  margin-top: 120px;
+  margin-bottom: 130px;
+}
+
+.interest {
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  font-style: normal;
+  width: 327px;
+  height: 216px;
+  text-align: center;
+  color: #172b4d;
+  display: contents;
+}
+
+.interest,
+h5 {
+  padding-right: 7px;
+  padding-left: 7px;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+  align-items: center;
+  justify-content: center;
+}
+
+input[type=text] {
+  background: transparent;
+  border: none;
+  border-bottom: 1px solid #000;
+  margin-top: 120px;
+  width: 300px;
+}
+
+input[type=email] {
+  background: transparent;
+  border: none;
+  border-bottom: 1px solid #000;
+  margin-top: 40px;
+  width: 300px;
+  margin-bottom: 40px;
+}
+
+textarea {
+  width: 303px;
+  height: 90px;
+  background: #fbf8f7;
+  font-size: 15px;
+  font-weight: 500;
+  padding: 1%;
+  margin-bottom: 24px;
```

Comment:
- Just as the previous reviewers have stated, you should please try to follow the details of the Figma design, as as to have a similar design to the Figma temple. 
   - You should remove the border on the contact textarea (The black line surrounding it) ad optionally add some padding to the text in the teaxtarea as it is in the Figma.
      - **Tip** You can do so using the CSS properties on the textarea class `border: none; padding: 15px;`
   - [Optionally] Please remove the black border on the button element and also use the proper font-family which is `Inter` currently you have `Arial` on your button text. **Tip** `border:none`

##### Your design
![image](https://user-images.githubusercontent.com/58771507/186335347-bdaf455e-0069-4230-9dd4-5a917c935b54.png)

##### Figma design
![image](https://user-images.githubusercontent.com/58771507/186335787-3c799f5e-b7b8-4bd0-bcf3-40ac3e6de628.png)